lib/tasks/status.rake

```diff
@@ -11,3 +11,18 @@ namespace :db do
     end
   end
 end
+
+namespace :redis do
+  desc 'Check for available redis connection'
+  task status: [:environment] do
+    begin
+      raise unless Redis.new(
```

Comment:
Observation: DRY?

Reply:
What is the recommendation? You don't like the `Redis.new(...)` part or the `begin/abort` logic? We don't ever use redis directly like this in any other place, since it's configured through sidekiq.

Reply:
Oh sorry, I should have been more specific. The connection parameters are being repeated. When we would change the normal usage within sidekiq initializer, then this has to change with it. Leave it as is, if you think it's not an issue. :slightly_smiling_face:

Reply:
Ok, agreed, this could be refactored in the future to be more DRY.